packages/mcp/README.md

```
@@ -0,0 +1,158 @@
+
+## Environment Variables
+
+| Variable | Required | Default | Description |
+|----------|----------|---------|-------------|
+| `RUSTRAK_API_TOKEN` | ✅ | — | 40-char hex API token. Server exits if missing. |
+| `RUSTRAK_API_URL` | — | `http://localhost:8080` | Base URL of your Rustrak server. |
+
+## Available Tools
+
+### Projects
+| Tool | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)